src/main/java/core/basesyntax/strategy/ReturnOperation.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax.strategy;
+
+import core.basesyntax.model.FruitTransaction;
+import java.util.Map;
+
+public class ReturnOperation implements OperationHandler {
+    @Override
+    public void handle(FruitTransaction transaction, Map<String, Integer> inventory) {
+        inventory.put(transaction.getFruit(),
+                inventory.getOrDefault(transaction.getFruit(), 0) + transaction.getQuantity());
+    }
```

Comment:
The `ReturnOperation` class correctly implements the `OperationHandler` interface. However, as per the checklist, you should consider checking the result balance before saving it in the Storage to ensure it remains positive. While the 'return' operation typically wouldn't result in a negative balance, maintaining this check across all operation handlers promotes consistency and guards against future changes that might introduce issues.